deepchem/data/datasets.py

```diff
@@ -636,9 +637,30 @@ def iterbatches(self,
                   epoch=0,
                   deterministic=False,
                   pad_batches=False):
-    """Get an object that iterates over minibatches from the dataset.
-
+    """ Get an object that iterates over minibatches from the dataset. It is guaranteed
+    that the number of batches returned is math.ceil(dataset.get_shape()[0][0]/batch_size).
```

Comment:
Maybe just say `len(dataset)/batch_size`. I believe this means the same thing and is more readable.

Reply:
Thanks - will fix this